tests/integration/targets/mso_schema_template_migrate/tasks/main.yml

```diff
@@ -0,0 +1,221 @@
+# Test code for the MSO modules
+# Copyright: (c) 2020, Anvitha Jain (@anvitha-jain) <anvjain@cisco.com>
+# 
```

Comment:
you may want to remove the empty line

Reply:
All the other modules are written in this format. @xinyuezhao @shrsr